spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -1433,6 +1434,45 @@ static void queryWithTimestampParameter(DatabaseClient dbClient) {
   }
   // [END spanner_query_with_timestamp_parameter]
 
+  // [START spanner_create_client_with_query_options]
+  static void clientWithQueryOptions(DatabaseId db) {
+    SpannerOptions options =
+        SpannerOptions.newBuilder()
+            .setDefaultQueryOptions(
+                db, QueryOptions.newBuilder().setOptimizerVersion("latest").build())
```

Comment:
I think here we want to set it to a version that's not the default so changing it to "1" would be ideal.
```suggestion
                db, QueryOptions.newBuilder().setOptimizerVersion("1").build())
```

Reply:
Done.